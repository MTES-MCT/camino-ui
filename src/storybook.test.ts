import initStoryshots, {
  Stories2SnapsConverter
} from '@storybook/addon-storyshots'
import { mount } from '@vue/test-utils'
import { resolve } from 'path'

initStoryshots({
  suite: 'Automated Storybook Snapshots',
  configPath: '.storybook',
  stories2snapsConverter: new Stories2SnapsConverter({
    snapshotExtension: '.storyshot',
    storiesExtensions: ['.js', '.ts', '.mdx']
  }),
  test: ({ story, context, stories2snapsConverter }) => {
    const snapshotFileName = resolve(
      stories2snapsConverter.getSnapshotFileName(context)
    )

    const storyElement = story.render()
    const wrapper = mount(storyElement)
    expect(wrapper.element).toMatchSpecificSnapshot(snapshotFileName)
  }
})
