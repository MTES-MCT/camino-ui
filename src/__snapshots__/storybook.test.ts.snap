// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Ui/Pill Error 1`] = `
<span
  class="cap-first small bold"
>
  <span
    class="bg-neutral color-bg pill py-xs px-s"
  >
    
    hello
    
  </span>
</span>
`;

exports[`Storyshots Ui/Pill Primary 1`] = `
<span
  class="cap-first small bold"
>
  <span
    class="bg-neutral color-bg pill py-xs px-s"
  >
    
    hello
    
  </span>
</span>
`;
